rules/no-disabled-tests.js

```diff
@@ -19,6 +19,33 @@ function getName(node) {
   return null;
 }
 
+function collectReferences(scope) {
```

Comment:
This function takes the current scope (so wherever the `pending()` function is called) and walks up the tree to collect all local references and [unresolved](https://eslint.org/docs/developer-guide/scope-manager-interface#through) references. This information is helpful for determining what local variables or functions have been defined in a file, as well as what variables that ESLint could not resolve (which likely means those are globals, like `describe()` & `test()`).